build_all.sh

```diff
@@ -83,6 +83,7 @@ targets="\
 	Allwinner|H6|aarch64|image \
 	Allwinner|R40|arm|image \
 	Amlogic|AMLGX|aarch64|image \
+        Ayn|Odin|aarch64|image \
```

Comment:
Remove space ?
```

diff --git a/build_all.sh b/build_all.sh
index a2fd7180bf..b50e119a1b 100755
--- a/build_all.sh
+++ b/build_all.sh
@@ -83,6 +83,7 @@ targets="\
 	Allwinner|H6|aarch64|image \
 	Allwinner|R40|arm|image \
 	Amlogic|AMLGX|aarch64|image \
+        Ayn|Odin|aarch64|image \
 	Generic||i386|image \
 	Generic||x86_64|image \
 	L4T|Switch|aarch64|image \
```

Reply:
Hmmm..... Need to look into that, as it lined up in my editor.....